src/material/icon/icon.spec.ts

```diff
@@ -1237,6 +1250,106 @@ describe('MatIcon without HttpClientModule', () => {
   });
 });
 
+describe('MatIcon with default options', () => {
+  it('should be able to configure color globally', fakeAsync(() => {
+    const fixture = createComponent(IconWithLigature, [
+      {provide: MAT_ICON_DEFAULT_OPTIONS, useValue: {color: 'accent'}},
+    ]);
+    const iconElement = fixture.debugElement.nativeElement.querySelector('mat-icon');
+    fixture.detectChanges();
+
+    expect(iconElement.classList.contains('mat-icon-no-color'))
+      .withContext('Expected the mat-icon element to not include the mat-icon-no-color class')
```

Comment:
It might be easier to read as `expect(iconElement.classList).not.toContain('mat-icon-no-color')`

I don't think the additional context messages provide anything additional and they can be removed

Reply:
thanks, done! 🙂